doc/integrator/docker.rst

```diff
@@ -80,3 +80,53 @@ You can also remove the unused named images, that should be done manually:
 
 * List the images with ``docker images``.
 * Remove the unwanted images with ``docker rmi <image>:<tag>``.
+
+
+Configure project for OpenShift
+-------------------------------
+
+This section is specific to projects using OpenShift for deployment, relying
+on Travis to build Docker images that are then pushed to Docker Hub.
```

Comment:
Shouldn't we add a note to informs about that can be done only by Camptocamp people?